circleci: failed tests (13504): TestPut

The following test appears to have failed:

[#13504](https://circleci.com/gh/cockroachdb/cockroach/13504):

```
I160229 16:00:28.841635 acceptance/replication_test.go:66  found 2 replicas
I160229 16:00:29.843953 acceptance/replication_test.go:66  found 2 replicas
I160229 16:00:30.846499 acceptance/replication_test.go:66  found 2 replicas
I160229 16:00:31.849286 acceptance/replication_test.go:66  found 2 replicas
I160229 16:00:31.950534 acceptance/cluster/localcluster.go:606  stopping
--- FAIL: TestPut (30.76s)
	testing.go:188: acceptance/replication_test.go:72: condition failed to evaluate within 20s: expected 3 replicas, only found 2
=== RUN   TestDockerPython
I160229 16:00:33.043743 acceptance/cluster/docker.go:72  ImagePull cockroachdb/docker-spy:20160209-143235 starting
I160229 16:00:33.790377 acceptance/cluster/docker.go:87  ImagePull cockroachdb/docker-spy:20160209-143235 complete
I160229 16:00:35.362635 acceptance/cluster/localcluster.go:262  started docker-spy: 172.17.0.50
I160229 16:00:35.362680 acceptance/cluster/localcluster.go:269  Initializing Cluster:
Cluster Setup: AdHoc 3x1
Duration: 5s, Stall: 2m0s
Node 0 - Store 0
Node 1 - Store 0
--
--- SKIP: TestBuildBabyCluster (0.00s)
	terraform_test.go:30: only enabled during testing
=== RUN   TestFiveNodesAndWriters
--- SKIP: TestFiveNodesAndWriters (0.00s)
	util_test.go:64: running in docker mode
FAIL
ok github.com/cockroachdb/cockroach/acceptance 1337s

```
Please assign, take a look and update the issue accordingly.

--- a/gossip/gossip.go
+++ b/gossip/gossip.go
@@ -768,6 +768,10 @@ func (g *Gossip) bootstrap() {
 					// Try to get another bootstrap address from the resolvers.
 					if addr := g.getNextBootstrapAddress(); addr != nil {
 						g.startClient(addr, stopper)
+					} else {
+						// We couldn't start a client, signal that we're stalled so that
+						// we'll retry.
+						g.maybeSignalStalledLocked()
 					}
 				}
 			})
@@ -777,14 +781,14 @@ func (g *Gossip) bootstrap() {
 			select {
 			case <-bootstrapTimer.C:
 				bootstrapTimer.Read = true
-				// continue
+				// break
 			case <-stopper.ShouldStop():
 				return
 			}
 			// Block until we need bootstrapping again.
 			select {
 			case <-g.stalled:
-				// continue
+				// break
 			case <-stopper.ShouldStop():
 				return
 			}

